lib/pure/httpcore.nim

```diff
@@ -100,16 +101,30 @@ const
 const httpNewLine* = "\c\L"
 const headerLimit* = 10_000
 
-proc newHttpHeaders*(): HttpHeaders =
+proc toTitleCase(s: string): string =
+  result = newString(len(s))
+  var upper = true
+  for i in 0..len(s) - 1:
+    result[i] = if upper: toUpperAscii(s[i]) else: toLowerAscii(s[i])
+    upper = s[i] == '-'
+
+
+proc newHttpHeaders*(titleCase: bool =false): HttpHeaders =
```

Comment:
you can just write "titleCase = false" here

Reply:
OK. Changed